src/Operation/Reject.php

```diff
@@ -37,30 +37,24 @@ public function __invoke(): Closure
              *
              * @return Closure(Iterator<TKey, T>): Generator<TKey, T>
              */
-            static fn (callable ...$callbacks): Closure =>
-                /**
-                 * @param Iterator<TKey, T> $iterator
-                 *
-                 * @return Generator<TKey, T>
-                 */
-                static function (Iterator $iterator) use ($callbacks): Generator {
-                    $defaultCallback =
-                        /**
-                         * @param T $value
-                         */
-                        static fn ($value): bool => (bool) $value;
-
-                    $callbacks = [] === $callbacks ?
-                        [$defaultCallback] :
-                        $callbacks;
-
-                    foreach ($iterator as $key => $current) {
-                        $result = CallbacksArrayReducer::or()($callbacks, $current, $key, $iterator);
-
-                        if (false === $result) {
-                            yield $key => $current;
-                        }
-                    }
-                };
+            static function (callable ...$callbacks): Closure {
```

Comment:
I can't tell for sure but does this closure have doc block type hints for the callables?

Reply:
The Filter's callback? If this is it, indeed, I forgot the type hints, I will fix later, after family lunch :)